### ISSUE TITLE
Add aXe (accessibility) tests

### ISSUE THREAD

**Issue by Terreii** [Terreii/andromeda-viewer#85]
**Is your feature request related to a problem? Please describe.**
There are at the moment no tests for accessibility!

**Describe the solution you'd like**
Add aXe tests! One solution could be [jest-axe](https://www.npmjs.com/package/jest-axe).

**Describe alternatives you've considered**
Manual testing and the aXe developer browser extension ([Firefox](https://addons.mozilla.org/de/firefox/addon/axe-devtools/) and [Chrome](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=de&gl=DE&pageId=116958493533771099880)).

**Additional context**
A web app should be accessible! It would be helpful if some tests would run automatically!
